modules/microshift-restoring-data-backups.adoc

```diff
@@ -0,0 +1,36 @@
+//Module included in the following assemblies:
+//
+// * microshift_updating/microshift-update-options.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-restoring-data-backups-manually_{context}"]
+= Restoring application data backups manually
+
+You can restore application data from a backup manually. Backups can be restored after updates, or after other system events that remove or damage required data. Backups are in the `/var/lib/microshift-backups` directory by default. When restoring a backup, you must use the entire file path.
+
+[NOTE]
+====
+RPM-OSTree systems back up and restore data automatically.
+====
+
+.Prerequisites
+* Root access to the host.
+* Full path of the data backup file.
+* {product-title} is stopped.
+
+.Procedure
+. Manually restore an application backup file using the full file path of the backup you want to restore by running the following command:
++
+[source,terminal]
+----
+$ sudo microshift restore /var/lib/microshift-backups/<my_backup_full_file_path>
```

Comment:
To me "full file path" means "full absolute filepath" like `/var/lib/custom-backups/custom`, so interpreting this very literally we get: `/var/lib/microshift-backups/var/lib/custom-backups/custom`
```suggestion
$ sudo microshift restore /var/lib/microshift-backups/<my_backup>
```